docs/graphql/manual/schema/relationships/architecture.rst

```diff
@@ -0,0 +1,33 @@
+.. meta::
+   :description: Security, performance, schema integrity, and data federation with Hasura remote joins
+   :keywords: hasura, docs, data federation, remote relationship, remote join, remote schema
+
+.. _remote_joins_architecture:
+
+Architecture of remote relationships
```

Comment:
Let's call it `Architecture of Remote Joins`

Reply:
Lol that's what I had originally, cc @marionschleifer @rikinsk